programs/vote/src/vote_instruction.rs

```diff
@@ -170,15 +174,13 @@ pub fn update_node(
 ) -> Instruction {
     let account_metas = vec![
         AccountMeta::new(*vote_pubkey, false),
+        AccountMeta::new_readonly(*node_pubkey, false),
```

Comment:
Since `node_pubkey` is passed via the keys list, we can set `is_signer` to `true` here and drop the `with_signer()`

Reply:
ah, sgtm